ios/BlurView.mm

```diff
@@ -131,6 +131,7 @@ - (void)setReducedTransparencyFallbackColor:(nullable UIColor *)reducedTranspare
 
 - (UIBlurEffectStyle)blurEffectStyle
 {
+  if ([self.blurType isEqual: @"transparent"]) return UIBlurEffectStyleDark;
```

Comment:
Thank you for your good contribution on important issues.
I also want this feature with white background, so I used your changed code and just fixed this line as 
`if ([self.blurType isEqual: @"transparent"]) return UIBlurEffectStyleLight;`.
However it shows dark background and not working.. Do you have any idea about this problem?

Reply:
Hm. I've just tested both cases (dark and light backgrounds) and everything seems to be working great. No changes in the native code 🤔

Light theme/white background:
| 1 | 2 |
| ----------- | ----------- | 
| <img width="400" alt="1" src="https://github.com/Kureev/react-native-blur/assets/17552441/432f6d99-022a-4abf-a0e6-4823de2c26d5"> | <img width="400" alt="2" src="https://github.com/Kureev/react-native-blur/assets/17552441/0ec0fe0b-0460-4a1e-a9e0-ee81dc07d779"> |

Code is just like this. It doesn't depend on theme or whatever:
```
<BlurView
  blurAmount={7} 
  blurType={"transparent"} 
/>
```

Please make sure you're using it right. If the problem still exists, could you elaborate a bit? What kind of effect are you trying to achieve? The image will do